Misc/NEWS.d/next/Documentation/2022-04-23-03-20-18.gh-issue-91795.elzdMM.rst

```diff
@@ -0,0 +1,2 @@
+Updated build_pgo part under 'Profile Guided Optimization' according to the
+current build script. Updated Microsoft docs link.
```

Comment:
Don't need a NEWS entry for this. Delete the file and I'll add the tag to skip the check